install/installer/pkg/components/ide-service/ide_config_configmap.go

```diff
@@ -110,6 +110,18 @@ func ideConfigConfigmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 					ImageLayers:       []string{codeWebExtensionImage, codeHelperImage},
 					LatestImage:       resolveLatestImage(ide.CodeIDEImage, "nightly", ctx.VersionManifest.Components.Workspace.CodeImage),
 					LatestImageLayers: []string{codeWebExtensionImage, codeHelperImage},
+					Versions: []ide_config.IDEVersion{
+						{
+							Version:     "1.87.0",
+							Image:       ctx.ImageName(ctx.Config.Repository, ide.CodeIDEImage, ide.CodeIDEImageStableVersion),
+							ImageLayers: []string{codeWebExtensionImage, codeHelperImage},
+						},
+						{
+							Version:     "1.85.2",
```

Comment:
It looks weird that we have 1.87 and 1.85.2, we should all intermediate then, it should be continuous. Or let's use 1.86 then instead.